presto-docs/src/main/sphinx/release/release-321.rst

```diff
@@ -0,0 +1,61 @@
+===========
+Release 321
+===========
+
+General Changes
+---------------
+
+* Fix incorrect result of ``round`` function when applied to ``tinyint``, ``smallint``, ``integer``
+  or ``bigint`` type with negative decimal places. (:issue:`42`)
+* Load data lazily in connector ``information_schema`` tables. This highly improves
+  performance for queries like: ``SELECT ... FROM catalog.information_schema.columns ... LIMIT ...```. (:issue:`1543`)
+
+Security Changes
+----------------
+
+* Improve query results authorization security with one-time tokens. (:issue:`1654`)
+
+Hive Connector Changes
+----------------------
+
+* Fix reading ``TEXT`` file collection delimiter set by Hive version earlier
+  than 3.0. (:issue:`1714`)
+* Fix AWS Glue support broken in Presto 320. (:issue:`1698`)
+* Improve performance for broadcast joins by using dynamic filtering.
+  This can be enabled via ``experimental.enable-dynamic-filtering``
+  configuration property (:issue:`1686`).
+* Support header and footer skip line count for ``CSV`` table with ``skip_header_line_count``
+  and ``skip_footer_line_count``. (:issue:`1090`)
+* Rename table property ``textfile_skip_header_line_count`` to ``skip_header_line_count``
+  and rename ``textfile_skip_footer_line_count`` to ``skip_footer_line_count``. (:issue:`1090`)
+* Add support for LZOP-compressed (``.lzo``) files. Previously a query accessing LZOP-compressed files
```

Comment:
```
* Add support for LZOP compressed (``.lzo``) files. Previously, queries accessing LZOP compressed
  files would fail, unless all files were small. (:issue:`1701`)